### DISCUSSION TITLE
Use Teleport with Ansible

### DISCUSSION THREAD

**Discussion by egulpen2** [gravitational/teleport#14835] [Q&A]
Hi Everyone,

I would like to use Teleport with Ansible.
At the moment we have a jumphost server that we use for our Ansible operations. I would like to use Teleport for that in the future.

I tried to configure it using the manual but i cannot make it work. Does anyone have a quick guide or steps to make this work?
I'm using the Community Edition.

Thanks in advance!

**Comment by webvictim** [comment]
Take a look at this guide: https://github.com/gravitational/teleport/blob/branch/v8/docs/pages/server-access/guides/ansible.mdx

(this is currently not visible in the docs, but should help for a human user)

**Reply by egulpen2** [reply]
Thanks for this guide!
I set up the absible.cfg and ssh.cfg only when trying to connect i get the following error:

ssh: Could not resolve hostname XXXXXXXX: Temporary failure in name resolution

I can understand that the Ubuntu system cannot resolve the name of the host just by hostname. This should go through the TSH agent i guess?

Any idea how to resolve this? Using tsh ssh root@XXXX works perfect.

Thanks!

Eric

**Comment by dgiorgio** [comment]
I had the same problem, and I was able to make it work.
Look at this discussion, if it helps you.
https://github.com/gravitational/teleport/discussions/13885
